src/wasm-interpreter.h

```diff
@@ -296,8 +296,9 @@ class ExpressionRunner : public Visitor<SubType, Flow> {
       case ReinterpretFloat64:   return value.castToI64();
       case DemoteFloat64:        return value.demote();
 
-      default: WASM_UNREACHABLE();
+      case InvalidUnary: break;
```

Comment:
seems nicer to put the WASM_UNREACHABLE on this line, instead of after the switch

happens a bunch more places below too

Reply:
Unfortunately this pattern is necessary to keep the older compilers from complaining about reaching the end of a non-void function.

Reply:
I see. How about putting a WASM_UNREACHABLE in both places, then? Not sure what I prefer better, but the break seems like it could be misread in some cases.